README.md

```diff
@@ -16,13 +16,12 @@ git clone https://github.com/chinedufn/webgl-water-tutorial
 cd webgl-water-tutorial
 
 # A version of Rust that can compile wasm-bindgen-cli version 0.2.29
-rustup override set nightly-2021-02-11
-cargo +nightly-2020-06-22 install -f wasm-bindgen-cli --version 0.2.29 # Or download a release binary
+cargo +$(cat rust-toolchain) install -f wasm-bindgen-cli --version 0.2.29 # Or download a release binary
```

Comment:
I think the `rust-toolchain.toml` file would mean that we no longer need to specify `+(... get nightly version ...)` here?

Reply:
@chinedufn thanks for suggestion, also added ~/.cargo/config for default target